db/migrate/20190821201800_swap_category_to_relation.rb

```diff
@@ -0,0 +1,25 @@
+class SwapCategoryToRelation < ActiveRecord::Migration[5.0]
```

Comment:
Not super important, but the filename is confusing to me. Maybe it should be ``SwapSongCategoryToAssociation`` or something along those lines.